homework_9/worker.py

```diff
@@ -0,0 +1,54 @@
+# 2. Create a class with a description of the worker. Any worker. employees.
+# I will evaluate the completeness of the described class and will withdraw points for everything unnecessary.
+# I want to see clean code. I expect to see clean code with annotations.
+# So far, no first-class connections with the second.
+# In all methods, I expect to see docstrings with a sane description.
+
+from random import randint
+
+
+class Worker:
+
+    def __init__(self, name: str, surname: str, position: str, salary: float):
+        self.__name = name
+        self.__surname = surname
+        self.__position = position
+        self.__salary = salary
+
+    @property
+    def name(self):
+        return self.__name
+
+    @property
+    def surname(self):
+        return self.__surname
+
+    @property
+    def position(self):
+        return self.__position
+
+    def calculate_new_salary(self):
+        performance = randint(1, 100)
+        self.__salary *= 1 + performance / 100
+        return self.__salary, performance
+
+    def annual_bonus(self):
+        if self.__position == 'Employee':
+            return 1000
+        elif self.__position == 'Lead':
+            return 2000
+        else:
+            return 3000
+
+    def salary_info_for_worker(self):
```

Comment:
General comment about function naming. It is good practice to start the name of the function with  verb.
For example:
get_salary_info....
calculate_anual_bonus..